tests/testthat/test-fotmob.R

```diff
@@ -4,9 +4,14 @@ context("Testing fotmob functions")
 test_that("fotmob_get_matches_by_date() works", {
   testthat::skip_on_cran()
 
-  results <- fotmob_get_matches_by_date(date = c("20210925", "20210926"))
```

Comment:
I was getting no results for these dates... I think Fotmob stops availability of matches that happened more than a year ago from the current date.

Reply:
Yeah interesting. At some stage we may look to make these static dates update automatically if we see it happening, but that might add extra layers of complexity unnecessarily

Reply:
@tonyelhabr, looks like this function failed R-CMD-check https://github.com/JaseZiv/worldfootballR/actions/runs/3223562730/jobs/5273776196#step:8:2268

Reply:
I tried 50 different days an realized that this function is inconsistent with how many columns it returns. e.g. `match_status_awarded` appeared in 7, `match_status_aggregated_str` appeared in 18, `away_pen_score` and `home_pen_score` in 31, etc. Nonetheless, a majority of columns still appeared in all 50 tries.

This is due to the `unnest()`ing of all columns in the JSON (instead of `select`ing columns explicitly), and the JSON having different fields depending on matches that occurred on the given date.

I think the solution is to just check for columns that will always be in the results.

Reply:
Just noticed this commit also ended in a failed check?

Reply:
should be good now